<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>center6</class>
 <widget class="QWidget" name="center6">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>52</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>120</width>
    <height>52</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>120</width>
    <height>52</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string>行動是治癒恐懼的良藥，而猶豫、拖延將不斷滋養恐懼</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#center6 {
background-color: rgba(255, 255, 255, 0);
}</string>
  </property>
  <widget class="QLabel" name="memUse">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>52</width>
     <height>52</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>52</width>
     <height>52</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font: 700 13pt &quot;Magneto&quot;;
color: rgb(135, 243, 255);
border-color: rgb(148, 148, 148);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>5</y>
     <width>68</width>
     <height>42</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>68</width>
     <height>42</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>68</width>
     <height>42</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frame{
	background-color: rgba(215, 255, 130, 150);
	border-top-right-radius: 4px;
    border-bottom-right-radius: 4px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QLabel" name="netUp">
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Microsoft YaHei UI&quot;;
color: rgb(252, 255, 96);
color: rgb(255, 121, 44);
background-color: transparent;
</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="netDown">
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Microsoft YaHei UI&quot;;
color: rgb(48, 255, 124);
background-color: transparent;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="memColorFrame">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>44</width>
     <height>44</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="borderColor">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>52</width>
     <height>52</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(32, 255, 188, 100);
border-radius: 5px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>borderColor</zorder>
  <zorder>memColorFrame</zorder>
  <zorder>memUse</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
